erpnext/projects/doctype/project/project.js

```diff
@@ -16,6 +16,54 @@ frappe.ui.form.on("Project", {
 					time_log.parenttype = 'Timesheet';
 					new_doc.time_logs = [time_log];
 
+					frappe.ui.form.make_quick_entry(doctype, null, null, new_doc);
```

Comment:
@hrwX can you move this code to a generalised function and  use it for all the doctypes.

Reply:
okay will do that